src/GraphQL/Utilities/Visitors/SchemaValidationVisitor.cs

```diff
@@ -61,7 +61,7 @@ public override void VisitObjectFieldDefinition(FieldType field, IObjectGraphTyp
                 throw new InvalidOperationException($"The field '{field.Name}' of an Object type '{type.Name}' has '{nameof(GraphQLTypeReference)}' type. This type must be replaced with a reference to the actual GraphQL type before using the reference.");
 
             // 2.3
-            if (!field.ResolvedType.IsOutputType())
+            if (!field.ResolvedType!.IsOutputType())
```

Comment:
Why changed this file? I have not seen `FieldType.ResolvedType` to change.

Reply:
Just cleaning up some build warnings probably

Reply:
ok